source/ohhako/boj/bj_1010.java

```diff
@@ -0,0 +1,28 @@
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.BufferedReader;
+
+public class Main{
+    public static void makeBridge(int n,int m){
+        int answer=0;
+        int parent=1;
+        int child=1;
+        for(int i=0 ; i<n-1 ; i++){
+            parent = parent*(m-i);
+        }
+        parent = parent*(m-n+1);
+        
+        for(int i=1 ; i<=n ; i++)
+            child = child*i;
```

Comment:
i have to use BigInteger because variable `parent`,`child` can't handle the value.